operations/admin.go

```diff
@@ -212,3 +214,56 @@ func doViewSettings() cli.ActionFunc {
 		return nil
 	}
 }
+
+func updateSettings() cli.Command {
+	const updateFlagName = "update"
+
+	return cli.Command{
+		Name:  "update-settings",
+		Usage: "update the evergreen configuration settings",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  joinFlagNames(updateFlagName, "u"),
+				Usage: "update to make (JSON or YAML)",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			confPath := c.Parent().String(confFlagName)
+			updateString := c.String(updateFlagName)
+			if updateString == "" {
+				return errors.New("no update string specified")
+			}
```

Comment:
does it make sense to put the parsing in a before function

Reply:
yep, moved